src/kOS/Module/kOSProcessor.cs

```diff
@@ -132,6 +130,26 @@ public Screen.TermWindow GetWindow()
 
         public override void OnStart(StartState state)
         {
+			//Populate selector for boot scripts
```

Comment:
Would we want to do this if we are not in the editor? Also would you mind moving it into its own method?: